Config.h

```diff
@@ -6,6 +6,72 @@
 
 constexpr auto KEYMOD_NONE = 0;
 
+constexpr bool DEFAULT_FULLSCREEN_MODE = false;
+constexpr int DEFAULT_RESOLUTION_WIDTH = 800;
+constexpr int DEFAULT_RESOLUTION_HEIGHT = 600;
+constexpr int DEFAULT_SIZE = /*20*/ 70;
+constexpr int DEFAULT_SIZE_Z = /*6*/ 4;
+
+class Config {
+    // constexpr auto GFXMODE = GFX_AUTODETECT_WINDOWED;
+public:
+    enum trackingmode : uint8_t {
+        TRACKING_NONE,
+        TRACKING_CENTER,
+        TRACKING_FOCUS,
+
```

Comment:
extra space?

Reply:
that's how it was in the [original](https://github.com/DFHack/stonesense/blob/e4bf6dcd27b77a85a6b9a0fbc7a55d346ebc1620/commonTypes.h#L180) 🤷‍♀️